apps/cards/app.js

```diff
@@ -14,6 +14,22 @@
 Bangle.loadWidgets();
 Bangle.drawWidgets();
 
+// get brightness
+let brightness;
+
+function loadBrightness() {
+  try {
+      const d = require('Storage').readJSON("setting.json", 1) || {};
+      brightness = Object.assign({
+          'brightness': 0.1
+      }, d || {});
+      return d;
+  } catch(e){
```

Comment:
What exceptions were you seeing here? Was it from an older version of the code? `readJSON` won't through with `1` as the second argument, and nothing else throws in this code block

Reply:
Not a problem, sounds like we can drop the try-catch :)